src/app/public/modules/flyout/types/flyout-config.ts

```diff
@@ -6,18 +6,91 @@ import {
   SkyFlyoutPermalink
 } from './flyout-permalink';
 
+/**
+ * Specifies the configuration options to set up a flyout.
+ */
 export interface SkyFlyoutConfig {
+
+  /**
+   * Sets the flyout's `aria-describedby` attribute to support accessibility.
+   * The value should be the HTML element ID (without the leading `#`) of the element
+   * that describes the flyout. Typically, the description includes text on the flyout
+   * but not on items that users interact with such as buttons and forms.
+   */
   ariaDescribedBy?: string;
+
+  /**
+   * Sets the flyout's `aria-labelledby` attribute to support accessibility.The value
+   * should be the HTML element ID (without the leading `#`) of the element that labels the flyout.
+   */
   ariaLabelledBy?: string;
+
+  /**
+   * Specifies an ARIA role for the flyout to support accessibility by indicating how
+   * the flyout functions and what it controls. The ARIA role indicates what
+   * the flyout represents on the web page. For information about ARIA roles,
+   * see the [WAI-ARIA roles model](https://www.w3.org/WAI/PF/aria/roles).
+   */
   ariaRole?: string;
+
+  /**
+   * Specifies the default width of the flyout container. If you do not provide a width,
+   * the flyout defaults to half the width of its container.
+   */
   defaultWidth?: number;
+
+  /**
+   * Specifies the minimum resize width of the flyout container.
+   * Defaults to `320`.
```

Comment:
I'm pretty sure we should delete the last sentence and add `@default 320` on another line.

That said, the current docs indicate `320` as the default. I assume that's pixels. If so, should we add `px` to the default value?

Reply:
Done. Its truly a number value, so we don't want `px` in this case.